packages/desktop-client/src/components/reports/reports/Spending.tsx

```diff
@@ -156,169 +170,228 @@ export function Spending() {
-            style={{
-              paddingRight: 5,
-              paddingLeft: 5,
-            }}
-          >
-            to
-          </Text>
-          <Select
-            value={compareTo}
-            onChange={e => {
-              setCompareTo(e);
-            }}
-            options={allIntervals.map(({ name, pretty }) => [name, pretty])}
-            disabled={mode !== 'singleMonth'}
-          />
-        </View>
         {!isNarrowWidth && (
           <View
             style={{
-              width: 1,
-              height: 30,
-              backgroundColor: theme.pillBorderDark,
-              marginRight: 15,
-              marginLeft: 10,
-            }}
-          />
-        )}
-        <View
-          style={{
-            flexDirection: 'row',
-            marginRight: 5,
-            marginTop: 5,
-            marginBottom: 5,
-          }}
-        >
-          <ModeButton
-            selected={mode === 'singleMonth'}
-            style={{
-              backgroundColor: 'inherit',
-            }}
-            onSelect={() => setMode('singleMonth')}
-          >
-            Single month
-          </ModeButton>
-          <ModeButton
-            selected={mode === 'budget'}
-            onSelect={() => setMode('budget')}
-            style={{
-              backgroundColor: 'inherit',
-            }}
-          >
-            Budgeted
-          </ModeButton>
-          <ModeButton
-            selected={mode === 'average'}
-            onSelect={() => setMode('average')}
-            style={{
-              backgroundColor: 'inherit',
-            }}
-          >
-            Average
-          </ModeButton>
-        </View>
-        {!isNarrowWidth && (
-          <View
-            style={{
-              width: 1,
-              height: 30,
-              backgroundColor: theme.pillBorderDark,
-              marginRight: 10,
-            }}
-          />
-        )}
-        <View
-          style={{
-            alignItems: 'center',
-            flexDirection: 'row',
-            marginBottom: 5,
-            marginTop: 5,
-            flex: 1,
-          }}
-        >
-          <FilterButton
-            onApply={onApplyFilter}
-            compact={isNarrowWidth}
-            hover={false}
-            exclude={['date']}
-          />
-          <View style={{ flex: 1 }} />
-          <Tooltip
-            placement="top end"
-            content={<Text>Save compare and filter options</Text>}
-            style={{
-              ...styles.tooltip,
-              lineHeight: 1.5,
-              padding: '6px 10px',
-              marginLeft: 10,
+              flexDirection: 'row',
+              alignItems: 'center',
+              flexShrink: 0,
             }}
           >
             <Button
-              variant="primary"
+              variant={isLive ? 'primary' : 'normal'}
+              onPress={() => setIsLive(state => !state)}
+            >
+              {isLive ? 'Live' : 'Static'}
```

Comment:
Looks like you've translated everything but this one, just highlighting in case you missed it.

Reply:
Good catch: https://github.com/actualbudget/actual/pull/3432/commits/4a8892810329a136d3c398d7cd84fd51c0b16f01